### ISSUE TITLE
[Accessibility] [Sample] ExpenseReport.JSON: Visually appearing string have insufficient colo contrast under ExpenseReport.JSON.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6268]
### Target Platforms

Other

### SDK Version

Main

### Application Name

All

### Problem Description

[23713889](https://microsoft.visualstudio.com/OS/_workitems/edit/23713889)

Elements doesn't have sufficient color contrast(3.1.:1) for visually appearing string 'Pending' and due to this, visual impaired user is unable to see the text.

### Card JSON

[ExpenseReport.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.2/Scenarios/ExpenseReport.json)
